corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap5/hq.helpers.js

```diff
@@ -3,11 +3,13 @@ hqDefine("hqwebapp/js/bootstrap5/hq.helpers", [
     'knockout',
     'underscore',
     'analytix/js/google',
+    'es6!hqwebapp/js/bootstrap5_loader',
```

Comment:
What does the `es6!` prefix mean/do here?

Reply:
It tells requirejs that this dependency uses ES6 syntax and should be read using a babel plugin. Context here: https://github.com/prantlf/requirejs-babel?tab=readme-ov-file#babel-plugin-for-requirejs (that link is to a fork, but the README is more extensive than on the original repo).